### DISCUSSION TITLE
Is there a way to get a list of all accounts using "swell-js" library?

### DISCUSSION THREAD

**Discussion by dragn-dev** [swellstores/community#532] [Help] [answered]
I'm trying out something like this.

![image](https://github.com/swellstores/community/assets/81596450/1f29725e-bb3c-4f05-ae64-94a28d60305e)

Basically when the user enters an email in my application I want to check if the account exists before showing the next action. But the only response I get is: 

![image](https://github.com/swellstores/community/assets/81596450/cbe44d9f-41ea-4807-96f1-6bb4984167c1)

Is there a better way to do this or am I doing something wrong?

**Comment by swellmike** [comment, marked as answer]
The swell-js library is a client for Swell's [Frontend API](https://developers.swell.is/frontend-api/introduction), which is concerned with the client-side (frontend) shopping experience.  As such, the permissions are restricted to an individual session's shopping experience (ie, you can access your own account, but not others').

Accessing all accounts (or fetching an account by ID) is done thru Swell's [Backend API](https://developers.swell.is/backend-api/introduction).  These actions are meant to be run server-side and require secret key authorization.  You can use the [swell-node](https://www.npmjs.com/package/swell-node) library or roll your own API client to access the Backend API.
